Makefile

```diff
@@ -11,7 +11,7 @@ clean: coverageclean typecoverageclean
 
 typecheck:
 	mypy --cobertura-xml-report typecover --html-report typecover .
-	mypy tests/integration
+	mypy tests
```

Comment:
It was helpful to do type checking in one of my unit tests, so I increased the generality here.